helm/values.yaml

```diff
@@ -171,6 +171,8 @@ configMap:
           # Drop spans that cannot be cured and will eventually end up being dropped anyway
           # after multiple retries.
           drop_spans: true
+        producer:
```

Comment:
I need to deprecate the `compression` config above and move `level` under `producer`.